generator/src/main/java/org/apache/kafka/message/MessageDataGenerator.java

```diff
@@ -612,7 +618,9 @@ private String readFieldFromStruct(FieldType type, String name) {
             return String.format("struct.getUUID(\"%s\")", name);
         } else if (type.isString()) {
             return String.format("struct.getString(\"%s\")", name);
-        } else if (type.isBytes()) {
+        } else if (type.isBytes() && useByteArray) {
+            return String.format("struct.getBytes(\"%s\")", name);
```

Comment:
Ditto here.

Reply:
Sorry, but I don't know what "Ditto" means :)

I assume, that you want `!useByteArray` be removed.
If yes, then fixed.